cliquet/tests/test_views_oauth.py

```diff
@@ -5,6 +5,22 @@
 from .support import BaseWebTest, unittest
 
 
+class DeactivatedViewsTest(BaseWebTest, unittest.TestCase):
+    def get_app_settings(self):
+        settings = super(DeactivatedViewsTest, self).get_app_settings()
+        settings['fxa-oauth.enabled'] = False
+        return settings
+
+    def test_login_view_is_not_available(self):
+        self.app.get('/fxa-oauth/login', status=404)
+
+    def test_params_view_is_not_available(self):
+        self.app.get('/fxa-oauth/params', status=404)
```

Comment:
We want to keep this one, because we have the scope list in here, but we want to strip it to just the scope.